core/tabs/system-setup/tab_data.toml

```
@@ -97,3 +97,9 @@ name = "Remove Snaps"
 description = "This script is designed to remove snap"
 script = "remove-snaps.sh"
 task_list = "RP"
+
+[[data]]
+name = "TTY Fonts"
+description = "This Script will set the default TTY font to Terminus size 32 Bold"
+script = "terminus-tty.sh"
+task_list = "RP"
```

```suggestion
task_list = "I PFM"
```

I for installation
PFM for privileged file modification
This script doesn't remove any packages.